src/PendingShortScheduleCommand.php

```diff
@@ -35,8 +36,12 @@ public function everySeconds(float $frequencyInSeconds = 1): self
         return $this;
     }
 
-    public function command(string $artisanCommand):self
+    public function command(string $artisanCommand): self
     {
+        if (class_exists($artisanCommand)) {
+            $artisanCommand = Container::getInstance()->make($artisanCommand)->getName();
```

Comment:
Would it also work by using the `app` function here?

Reply:
> Nice! Could you also update the readme with an example?

Sure thing, I have updated readme file.

Reply:
> Would it also work by using the `app` function here?

Yes, I have updated it